README.md

```diff
@@ -460,6 +494,25 @@ cygnusagent.sinks.mysql-sink.mysql_password = xxxxxxxxxxxx
 # how the attributes are stored, either per row either per column (row, column)
 cygnusagent.sinks.mysql-sink.attr_persistence = column
 
+# ============================================
+# OrionMongoSink configuration
+# sink class, must not be changed
+cygnusagent.sinks.mongo-sink.type = com.telefonica.iot.cygnus.sinks.OrionMongoSink
+# channel name from where to read notification events
+cygnusagent.sinks.mongo-sink.channel = mongo-channel
+# comma-separated list of FQDN/IP:port pairs where the MongoDB server runs
+cygnusagent.sinks.mongo-sink.mongo_uri = x1.y1.z1.w1:port1,x2.y2.z2.w2:port2,...
```

Comment:
I'd suggest the following wording:

```
# FQDN/IP:port where the MongoDB server tuns (standalone case) or comma-separated list of FQDN/IP:port pairs where the MongoDB replica set members run
```
In addition, in the caes of using a replica set, where the name of the replica set is specified?